docs/content/preview/yugabyte-cloud/cloud-admin/cloud-billing-profile.md

```diff
@@ -93,18 +92,18 @@ To download the invoice as a PDF, click the **Download** icon.
 
 ### Invoice summary
 
-To view invoice details, select an invoice in **Invoice History** to display its **Summary**.
+To view invoice details, select an invoice in **Invoice History**.
 
 To download the invoice as a PDF, click **Download Invoice**.
 
 **Summary** shows the following invoice details:
 
-- Total - the amount you owe in the billing period (or running total in the current billing period; Yugabyte updates the running total once a day).
-- Billing option - your billing plan (pay-as-you-go or subscription).
+- Plan, Price, and Billing option - your plan type and how you are charged (pay-as-you-go or subscription).
 - Billing period - the start and end date of the current billing period. Your first billing period starts the day you created your billing profile and ends on the last day of the month; subsequent periods start on the first day of the month and end on the last.
+- Total - the amount you owe in the billing period (or running total in the current billing period; Yugabyte updates the running total once a day).
```

Comment:
@ddhodge modify to 
'Total billable amount due - the amount you owe in the billing period net of remaining credits (or for active invoices, Running total in the current billing period which Yugabyte updates once a day).'